### ISSUE TITLE
Repo Profile 404 for Valid Repo URL

### ISSUE THREAD

**Issue by avoinea** [docker/hub-feedback#193]
I miss the quick filtering feature from the old theme. Also The pagination seems broken. Here for example https://hub.docker.com/r/eeacms/ you don't know that there are 6 pages, it seems to be only one, so why to press the next/prev buttons?

Also I linked this everywhere in the documentation https://hub.docker.com/u/eeacms/plone-instance/

Now, this link is broken.

**Comment by ChristopherBiscardi** [comment]
#36 is tracking the pagination issue.

#198 is tracking quick filters

I'm going to make this issue about the 404 for `https://hub.docker.com/u/eeacms/plone-instance/`, which is a valid URL that should not be 404ing.

**Comment by victor-yarema** [comment]
@ChristopherBiscardi I think that this was a bad idea to rename existing issue. I ran into "Invalid page not found for URLs that start with `https://hub.docker.com/u/`". I came here to report one and found this issue. When I read the initial comment from @avoinea first time I didn't notice that this is about 404 bug. Because it is not. The problem is that after rename the initial comment and new name don't match. I even wanted to ask you "did you manage to create an issue regarding 404?". :)
Later I have read carefully your comment and understood where I am.
I propose not to rename issues but simply always create new ones, refer from each new one to old one and close old one.
Now regarding the issue with invalid 404.
I have also observed that server returns 404 response for unauthenticated requests. I was able to see the page after I have signed up and logged in.
I was checking out page with URL https://hub.docker.com/u/valhalla/dashboard/ .
